### PR TITLE
Added automation hub to renovate

### PR DESCRIPTION
**Pull request by garethahealy**
renovate config to search automation hub.

FYI: the token requires manually updating as renovate lacks support https://github.com/renovatebot/renovate/discussions/31854 - but hopefully in the future, they'll create an API to do this

### CODE REVIEW

**Comment by bkaraoren** [comment]
@garethahealy thanks for taking care. But we have no secret on GitHub named `ANSIBLE_GALAXY_SERVER_AUTOMATION_HUB_TOKEN`, I assume it has to be `RH_AUTOMATION_HUB_TOKEN` which we normally have in use.

File: renovate.json [hunk 1]
```diff
@@ -3,5 +3,22 @@
...
+      "registryUrls": [
+        "https://console.redhat.com/api/automation-hub/content/published/"
+      ]
+    }
+  ],
+  "hostRules": [
+    {
+      "matchHost": "https://console.redhat.com/api/automation-hub/content/published/",
+      "hostType": "galaxy-collection",
+      "token": "{{ secrets.ANSIBLE_GALAXY_SERVER_AUTOMATION_HUB_TOKEN }}"
```
Comment (bkaraoren) [inline renovate.json:21]:
```suggestion
      "token": "{{ secrets.RH_AUTOMATION_HUB_TOKEN }}"
```

**Comment by garethahealy** [comment]
@bkaraoren token exists on renovate, not GH

**Comment by bkaraoren** [comment]
> @bkaraoren token exists on renovate, not GH

I see, I approve as it is. thanks.

**Review by mmtktl — APPROVED** [review]
LGTM

**Pull request merged**